ssl/ssl_rsa.c

```diff
@@ -639,7 +639,7 @@ static int use_certificate_chain_file(SSL_CTX *ctx, SSL *ssl, const char *file)
     if (ctx != NULL) {
         passwd_callback = ctx->default_passwd_callback;
         passwd_callback_userdata = ctx->default_passwd_callback_userdata;
-    } else {
+    } else if (ssl != NULL) {
```

Comment:
I think the code assumes that either you have an ssl, or a ctx, and that if you somehow get here with `ssl == NULL`, something else is wrong.

Reply:
I think it is ok to leave this as a false positive and remove the change.. Thanks.